src/main/asciidoc/jdbc.adoc

```diff
@@ -86,108 +84,115 @@ class ApplicationConfig {
 ----
 ====
 
-The just shown configuration class sets up an embedded HSQL database using the `EmbeddedDatabaseBuilder` API of spring-jdbc. We finally activate Spring Data JDBC repositories using the `@EnableJdbcRepositories`. If no base package is configured it will use the one the configuration class resides in.
+The configuration class in the preceding example sets up an embedded HSQL database by using the `EmbeddedDatabaseBuilder` API of `spring-jdbc`.
+We activate Spring Data JDBC repositories by using the `@EnableJdbcRepositories`.
+If no base package is configured, it uses the package in which the configuration class resides.
 
 [[jdbc.entity-persistence]]
-== Persisting entities
+== Persisting Entities
 
-Saving an Aggregate can be performed via the `CrudRepository.save(…)`-Method. If the Aggregate is a new Aggregate this will result in an insert for the Aggregate Root, followed by insert statments for all directly or indirectly referenced entities.
+Saving an aggregate can be performed with the `CrudRepository.save(…)` method.
+If the aggregate is new, this results in an insert for the aggregate root, followed by insert statements for all directly or indirectly referenced entities.
 
-If the Aggregate Root is _not new_ all referenced entities will get deleted, the Aggregate Root updated and all referenced entities will get inserted again.
+If the aggregate root is not new, all referenced entities get deleted, the aggregate root gets updated, and all referenced entities get inserted again.
```

Comment:
"new" is a specific state the instance is in.  This might be clear from context but I wonder if there is an established way to make that clear. It's what I tried with the emphasis in the original version.